### DISCUSSION TITLE
Responsive `label` in `AreaChart`

### DISCUSSION THREAD

**Discussion by 201flaviosilva** [recharts/recharts#3146] [Q&A] [answered]
Hey there,

I have a problem with my chart, the axis labels are not being positioned in the right place:

<img width="754" alt="Chart" src="https://bytebucket.org/201flaviosilva-labs/q-recharts-areachart-responsive-label/raw/b68781d0bae4a7cb433d1fa3f990d84bd81d8f98/Preview.png">

The `DATA` and `MINUTES` are under the chart ticks, how do I fix this?

```jsx
<XAxis
  label="DATE"
  name="date"
  dataKey="date"
  padding={{ left: 42, right: 42 }}
/>
<YAxis label="MINUTES" name="minutes" dataKey="minutes" />
```

Code: https://bitbucket.org/201flaviosilva-labs/q-recharts-areachart-responsive-label/

**Comment by ckifer** [comment, marked as answer]
Hi @201flaviosilva 

You can adjust the`Label` by defining it with an object rather than giving a value directly

https://codesandbox.io/s/recharts-areachart-label-forked-58er5g?file=/src/App.js

```jsx
label={{
      value: "MINUTES",
      angle: -90,
      position: "insideLeft"
}}
```

**Reply by 201flaviosilva** [reply]
Hey, thx for your nice help :)
